### PR TITLE
21051 - Update MHR tile description and remove beta tag

### PR DESCRIPTION
**Pull request by seeker25**
https://github.com/bcgov/entity/issues/21051

### CODE REVIEW

File: components/BCRegistriesAccount.vue [hunk 1]
```diff
@@ -150,18 +150,16 @@
...
-        </span>
         <v-card-text>
           <p>
-            In the beta version of the Manufactured Home Registry, you are currently able to:
+            In the the Manufactured Home Registry, you are able to:
           </p>
 
           <ul>
-            <li>Search for, and download reports for, manufactured homes and personal property
+            <li>Search and download reports for, manufactured homes and personal property
```
Comment (dimak1) [inline components/BCRegistriesAccount.vue:159]: Not sure if comma is needed here.

**Review by dimak1 — APPROVED** [review]
Looks good, just one comment.

File: components/BCRegistriesAccount.vue (hunk 1 above)
Comment (seeker25) [reply components/BCRegistriesAccount.vue:159]: removed

**Review by cameron-eyds — APPROVED** [review]
Looks good!

**Comment by seeker25** [comment]
Might need a node upgrade soon

File: components/BCRegistriesAccount.vue [hunk 2]
```diff
@@ -150,18 +150,16 @@
       <!-- Manufactured Home Registry -->
       <v-card v-if="showMhrFlag" elevation="2">
         <v-card-title>Manufactured Home Registry</v-card-title>
-        <span v-if="showBetaMhr" class="card-title-badge-container">
-          <div class="card-title-badge font-weight-bold pt-2">BETA</div>
-        </span>
         <v-card-text>
           <p>
-            In the beta version of the Manufactured Home Registry, you are currently able to:
+            In the the Manufactured Home Registry, you are able to:
```
Comment (seeker25) [inline components/BCRegistriesAccount.vue:155]: `the the`

**Pull request merged**